var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -15,10 +15,13 @@ class PyMorphio(PythonPackage):
     git = "https://github.com/BlueBrain/MorphIO.git"
     pypi = "morphio/MorphIO-3.3.2.tar.gz"
 
-    license("LGPL-3.0-or-later")
+    license("Apache-2.0")
```

Comment:
I think we should have 2 `license` directives, with a `when=` argument, because v3.3.6 is still LGPL

Reply:
Thank you for you review @alalazo 
I took your remark into account in 2b9274423415
Best